KinoPoiskAutomatedTests/PageObjects/HomePage.cs

```diff
@@ -1,15 +1,52 @@
-﻿using OpenQA.Selenium;
-using SeleniumExtras.PageObjects;
+﻿using System;
+using KinoPoiskAutomatedTests.WebDriver;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
 
 namespace KinoPoiskAutomatedTests.PageObjects
 {
-    public class HomePage
+    public class HomePage : DefaultPage
     {
-        private IWebDriver driver;
+        private IWebElement _myAccountButton => driver.FindElement(By.XPath("//button[contains(@class,'login')]"));
+        private By _exitButton = By.XPath("//*[text()='Выйти']");
+        private By _avatar = By.XPath("//div[contains(@class,'avatar')][1]");
+        private IWebElement _searchButton => driver.FindElement(By.XPath("//div[contains(@class,'search')]/button"));
+        private By _searchInput =  By.XPath("//input[contains(@class,'search')]");
 
-        [FindsBy(How = How.XPath, Using = "//a[contains(@href,'passport.yandex.ru/auth?')]")]
-        [CacheLookup]
-        public IWebElement MyAccount { get; set; }
-    
+        public void PressLogInButton()
+        {
+            _myAccountButton.Click();
+        }
+
+        public void PressLogOutButton()
+        {
+            WebDriverExtensions.FindElement(driver, _avatar, Data.WaitTime).Click();
+            WebDriverExtensions.FindElement(driver,_exitButton,Data.WaitTime).Click();
+        }
+
+        public bool IsExitExist()
```

Comment:
Not a mistake, but you could use lambda expression
   `     public bool IsExitExist() => driver.GetWait().Until(d => driver.FindElement(_exitButton).Enabled);`

Reply:
resolved